ipynb/android-wloads/Android_YoutTube_Tests.ipynb

```diff
@@ -0,0 +1,496 @@
+      "2016-03-24 11:54:09,393 INFO    :         Target - Loading custom (inline) test configuration\n",
+      "2016-03-24 11:54:09,393 INFO    :         Target - Devlib modules to load: ['cpufreq']\n",
+      "2016-03-24 11:54:09,394 INFO    :         Target - Connecting Android target [{'device': '00b1346f0878ccb1'}]\n",
+      "2016-03-24 11:54:09,899 INFO    :         Target - Initializing target workdir:\n",
+      "2016-03-24 11:54:09,901 INFO    :         Target -    /data/local/tmp/devlib-target\n",
+      "2016-03-24 11:54:13,212 INFO    :         Target - Topology:\n",
+      "2016-03-24 11:54:13,214 INFO    :         Target -    [[0]]\n",
+      "2016-03-24 11:54:14,625 INFO    :         FTrace - Enabled tracepoints:\n",
+      "2016-03-24 11:54:14,627 INFO    :         FTrace -   sched_switch\n",
+      "2016-03-24 11:54:14,627 INFO    :         FTrace -   sched_load_avg_cpu\n",
+      "2016-03-24 11:54:14,628 INFO    :         FTrace -   cpu_frequency\n",
+      "2016-03-24 11:54:14,628 INFO    :         FTrace -   cpu_capacity\n",
+      "2016-03-24 11:54:14,629 WARNING :        TestEnv - Wipe previous contents of the results folder:\n",
+      "2016-03-24 11:54:14,629 WARNING :        TestEnv -    /home/micgio01/work/lisa/results/AndroidN5X_Youtube\n",
+      "2016-03-24 11:54:14,738 INFO    :        TestEnv - Set results folder to:\n",
+      "2016-03-24 11:54:14,739 INFO    :        TestEnv -    /home/micgio01/work/lisa/results/AndroidN5X_Youtube\n",
+      "2016-03-24 11:54:14,739 INFO    :        TestEnv - Experiment results available also in:\n",
+      "2016-03-24 11:54:14,739 INFO    :        TestEnv -    /home/micgio01/work/lisa/results_latest\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Initialize a test environment using:\n",
+    "# the provided target configuration (my_target_conf)\n",
+    "# the provided test configuration   (my_test_conf)\n",
+    "te = TestEnv(target_conf=my_target_conf, test_conf=my_tests_conf)\n",
+    "target = te.target"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Support Functions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This set of support functions will help us running the benchmark using different CPUFreq governors."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "def set_performance():\n",
+    "    target.cpufreq.set_all_governors('performance')\n",
+    "\n",
+    "def set_powersave():\n",
+    "    target.cpufreq.set_all_governors('powersave')\n",
+    "\n",
+    "def set_interactive():\n",
+    "    target.cpufreq.set_all_governors('interactive')\n",
+    "\n",
+    "def set_sched():\n",
+    "    target.cpufreq.set_all_governors('sched')\n",
+    "\n",
+    "def set_ondemand():\n",
```

Comment:
This can be put to work with something like the following.

`def set_ondemand():
    target.cpufreq.set_all_governors('ondemand')
    
    for cpu in target.list_online_cpus():
        tunables = target.cpufreq.get_governor_tunables(cpu)
        target.cpufreq.set_governor_tunables(
            cpu,
            'ondemand',
            **{'sampling_rate' : tunables['sampling_rate_min']})`